coveralls/cli.py

```diff
@@ -86,8 +94,9 @@ def main(argv=None):
 
         log.info('Coverage submitted!')
         log.debug(result)
-        log.info(result['message'])
-        log.info(result['url'])
+        if result:
+            log.info(result.get('message', None))
+            log.info(result.get('url', None))
```

Comment:
nit: `dict.get` already returns `None` by default, no need to specify it.
```suggestion
            log.info(result.get('message'))
            log.info(result.get('url'))
```